python/paddle/tensor/manipulation.py

```diff
@@ -6861,3 +6861,67 @@ def slice_scatter(x, value, axes, starts, ends, strides, name=None):
         )
 
         return output
+
+
+def block_diag(*inputs, name=None):
```

Comment:
According to the current specification of PaddlePaddle, when the input may be multiple tensors, the parameters cannot be passed in this way. You need to put multiple tensors into a list or tuple before passing in. Please refer to the [broadcast_tensors API](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/broadcast_tensors_cn.html#broadcast-tensors)

Reply:
OK，please wait

Reply:
I have made the changes, please review @jeff41404